test/run_envoy_bazel_coverage.sh

```diff
@@ -18,32 +20,74 @@ if [[ $# -gt 0 ]]; then
 elif [[ -n "${COVERAGE_TARGET}" ]]; then
   COVERAGE_TARGETS=${COVERAGE_TARGET}
 else
-  COVERAGE_TARGETS=//test/...
+  # For fuzz builds, this overrides to just fuzz targets.
+  echo $(bazel --version)
+  COVERAGE_TARGETS=//test/... && [[ ${FUZZ_COVERAGE} ]] &&
+    COVERAGE_TARGETS="$(bazel query 'attr("tags", "fuzz_target", //test/...)')"
 fi
 
-# Make sure //test/coverage:coverage_tests is up-to-date.
 SCRIPT_DIR="$(realpath "$(dirname "$0")")"
-"${SCRIPT_DIR}"/coverage/gen_build.sh ${COVERAGE_TARGETS}
+TEMP_CORPORA=""
+if [ "$FUZZ_COVERAGE" == false ]
```

Comment:
I would build this logic around `FUZZ_COVERGE` as the positive case and always fallback to regular coverage, so that even if that variable is missing, it still works.